packages/desktop-client/src/components/modals/EditRule.js

```diff
@@ -397,13 +396,18 @@ function newInput(item) {
 
 export function ConditionsList({
   conditions,
-  conditionFields,
   editorStyle,
   isSchedule,
   onChangeConditions,
 }) {
   function addCondition(index) {
-    let field = 'payee';
+    // (remove the inflow and outflow pseudo-fields since they’d be a pain to get right)
+    let fields = conditionFields.slice(0, -2).map(f => f[0]);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏any other way how we could remove these specific fields?

I'm afraid that currently if someone decides to re-order the values in `conditionFields` we'll end up with a bug that's very hard to spot.